language/move-vm/types/src/gas_schedule.rs

```diff
@@ -277,6 +277,145 @@ pub fn zero_cost_schedule() -> CostTable {
     new_from_instructions(instrs, native_table)
 }
 
+pub static INITIAL_GAS_SCHEDULE: Lazy<CostTable> = Lazy::new(|| {
```

Comment:
Was the other one removed from vm_genesis? Maybe we can comment what this one is for?

Reply:
cc @tzakian

Reply:
I don't think this should exist here. This gas schedule is Diem-specific, so doesn't really make sense to be in the move-vm IMO. 

If you need a gas schedule I would recommend using either the zero-cost cost table, or a unit cost table (e.g., https://github.com/diem/diem/blob/main/language/tools/move-unit-test/src/test_runner.rs#L58).

Reply:
> I don't think this should exist here. This gas schedule is Diem-specific, so doesn't really make sense to be in the move-vm IMO.

That's reasonable. I did this because I didn't know better alternatives. BTW `pub enum NativeCostIndex` suffers from the same problem -- some of its entries are diem-framework natives. I think we should move it out of `move-vm-types`.

> If you need a gas schedule I would recommend using either the zero-cost cost table, or a unit cost table

The `unit-cost-table` sounds appealing to me. I guess it's good enough for the cli for testing purposes?

Reply:
The broader issue here is that native functions uses `NativeCostIndex` to determine their costs. This is problematic since these indices could be different for different deployments and different combinations of packages with native functions, at least in theory. In other words, I think we'll need to figure out a way to make the native cost table composable, or behave more like unordered sets in the future.